src/components/SelectLang/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { formatMessage, setLocale, getLocale } from 'umi/locale'
+import { Menu, Icon } from 'antd'
+import classNames from 'classnames'
+import HeaderDropdown from '../HeaderDropdown'
+import styles from './index.less'
+
+export default function SelectLang (props) {
+
+  function changeLang ({ key }) {
+    setLocale(key)
+  }
+
+  const { className } = props
+  const selectedLang = getLocale()
+  const langMenu = (
```

Comment:
this is static, so we can move it to the module scope (without exporting)

Reply:
- [ ] move `langMenu` to module scope

Reply:
done